terraform/api.tf

```diff
@@ -1,11 +1,11 @@
 resource "aws_lambda_function" "api" {
   description      = "API for ${local.pcc_api_name}"
-  function_name    = "${local.pcc_api_name}"
+  function_name    = local.pcc_api_name
   role             = aws_iam_role.lambda.arn
-  runtime          = "nodejs14.x"
+  runtime          = "nodejs18.x"
```

Comment:
I'm sure it's fine, but I think we're using node16 in our gh actions for deployment, I just wanted to make sure this is okay? I don't have enough context on the tf stuff to comment, but I just thought I'd mention it in case :)